Dockerfile-linux.template

```diff
@@ -392,6 +392,8 @@ RUN set -eux; \
 {{ ) else ( -}}
 		--enable-maintainer-zts \
 {{ ) end -}}
+# https://github.com/php/php-src/issues/9649#issuecomment-1264330874
```

Comment:
I think it would probably be useful for this to also link to https://externals.io/message/118859, where you've given a very nice summary of the situation (with a lot of additional links). :innocent:

Reply:
Link updated!